beacon/validator/types.go

```diff
@@ -237,3 +239,24 @@ type TelemetrySink interface {
 	// identified by the provided keys.
 	MeasureSince(key string, start time.Time, args ...string)
 }
+
+type BlobSidecars[T, BlobSidecarT any] interface {
+	constraints.Nillable
+	constraints.SSZMarshallable
+	constraints.Empty[T]
+	Len() int
+	Get(index int) BlobSidecarT
+	GetSidecars() []BlobSidecarT
+	ValidateBlockRoots() error
+	VerifyInclusionProofs(kzgOffset uint64) error
+}
+
+type BlockBuilderI[SlotDataT any] interface {
```

Comment:
@fridrik01 it seems to be the generic is not really needed here. Can we drop it?

Reply:
Good catch, also the same with BlockchainI